governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021
+summary: MIP39c2-SP24 adds Core Unit IS-001 - Immunefi Security
```

Comment:
"This MIP Set adds the Immunefi Security Core Unit (IS-001)"

Not really happy with that though, and we haven't been super consistent with these. Maybe "onboards" is better?

Reply:
Yeah I agree onboarding is better - I copied the wording from the DUX poll fwiw https://github.com/makerdao/community/blob/master/governance/polls/Ratification%20Poll%20for%20Development%20%26%20UX%20Core%20Unit%2C%20DUX-001%20-%20September%2013%2C%202021.md